### ISSUE TITLE
request for custom dsi menu

### ISSUE THREAD

**Issue by Kasun1218** [DS-Homebrew/twlmenu-extras#42]
[lemon demon.zip](https://github.com/DS-Homebrew/twlmenu-extras/files/8461146/lemon.demon.zip)

**Name**
   the lemon demon dsi menu

**Author**
   TWHD

**Description**
   i took some inspiration from the album covers and over all logo of the lemon demon band to create this menu for twilight menu++
icons are being worked on but all backgrounds and the sound and battery icons are complete

**Categories**
   music
   

**Icon**
![yeah yeah yeah](https://user-images.githubusercontent.com/96924957/162667195-dcccc887-3750-4526-80dd-9a967fe2af91.png)

**Screenshots**

**Comment by Kasun1218** [comment]
finished thanks for the wait 
[lemon demon.zip](https://github.com/DS-Homebrew/twlmenu-extras/files/8485251/lemon.demon.zip)

**Comment by Allinxter** [comment]
Is the box (`\grf\box.bmp`) supposed to look like this?
![imagen](https://user-images.githubusercontent.com/22733143/163316224-97b5fac9-8531-49a8-ae25-c79b7dbfe0ff.png)
It's shown as the raw file does, but it kinda looks like it's an exporting artifact

The text box (`\grf\dialogbox.bmp`), should not look like this, according to the raw files
![imagen](https://user-images.githubusercontent.com/22733143/163315660-2dd0f39b-df2a-42bb-80de-32f6bb7d3d16.png)

You also might want to use different colors for the Classic DS Menu background (`\quickmenu\)`, it's kinda hard to see the hour and buttom prompt
![imagen](https://user-images.githubusercontent.com/22733143/163317802-59713395-93a9-4b34-86bb-a1cde0ab325a.png)

**Comment by Kasun1218** [comment]
1 yes 
2 i dont know how to make the color consitent trough all colored themes
And 3 i can fix when i learn how to

**Comment by Epicpkmn11** [comment]
In `theme.ini` you can disable applying the user color palette with the following options:
```ini
StartTextUserPalette	= 1
StartBorderUserPalette	= 1
ButtonArrowUserPalette	= 1
MovingArrowUserPalette	= 1
LaunchDotsUserPalette	= 1
DialogBoxUserPalette	= 1
```
Each can be controlled individually, just change the `1` to `0` to disable it for that image.

**Comment by Kasun1218** [comment]
got the user pallete fixed how do i edit the ds menu  ui text color?

**Comment by Kasun1218** [comment]
Yes besides i wanna see how people do it this year to get an idea

**Comment by Kasun1218** [comment]
How do edit the ds mode font color

**Comment by Epicpkmn11** [comment]
added in 21ff97be6e78644d32fc6908c8a15908da5ad412
